x/ref/runtime/internal/rpc/testutil_test.go

```diff
@@ -7,7 +7,7 @@ package rpc
 import (
 	"time"
 
-	"v.io/v23"
+	v23 "v.io/v23"
```

Comment:
Are these local package names necessary?  Shouldn't the name of the package be `v23` anyways?

Reply:
this is just goimports fixing the ambiguity between go module version #s and v23 which is a valid version number!